From d89f60bd3468977f04b3563c0ebf82ca5b7b0910 Mon Sep 17 00:00:00 2001
From: Robin Burchell <robin+git@viroteck.net>
Date: Thu, 11 Jul 2013 08:53:06 +0000
Subject: [PATCH 2/7] fix documentation path

---
 lib/SignOn/doc/doc.pri | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/lib/SignOn/doc/doc.pri b/lib/SignOn/doc/doc.pri
index 46f1148..5c5da07 100644
--- a/lib/SignOn/doc/doc.pri
+++ b/lib/SignOn/doc/doc.pri
@@ -58,7 +58,11 @@ include( ../../../common-installs-config.pri )
 #-----------------------------------------------------------------------------
 # Installation target setup for documentation
 #-----------------------------------------------------------------------------
-documentation.path = $${INSTALL_PREFIX}/share/doc/libsignon-qt
+greaterThan(QT_MAJOR_VERSION, 4) {
+    documentation.path = $${INSTALL_PREFIX}/share/doc/libsignon-qt5
+} else {
+    documentation.path = $${INSTALL_PREFIX}/share/doc/libsignon-qt
+}
 for( folder, DOC_FOLDERS ) {
     documentation.files += $${folder}
 }
-- 
2.20.1

